command/server_util.go

```diff
@@ -14,63 +13,80 @@ var (
 	onEnterprise = false
 )
 
-func adjustCoreForSealMigration(ctx context.Context, core *vault.Core, coreConfig *vault.CoreConfig, seal vault.Seal, config *server.Config) error {
+func adjustCoreForSealMigration(ctx context.Context, core *vault.Core, barrierSeal, unwrapSeal vault.Seal) error {
```

Comment:
FWIW, `ctx` is passed in here but never used.

Reply:
Done.